embeddings/pipeline/lightning_pipeline.py

```diff
@@ -13,16 +19,51 @@ class LightningPipeline(
     Pipeline[EvaluationResult],
     Generic[Data, ModelResult, EvaluationResult],
 ):
+    DEFAULT_TASK_TRAIN_KWARGS: Dict[str, Any] = {"devices": "auto", "accelerator": "auto"}
+    DEFAULT_TASK_MODEL_KWARGS: Dict[str, Any] = {"use_scheduler": True}
+    DEFAULT_DATAMODULE_KWARGS: Dict[str, Any] = {"max_seq_length": None}
+    DEFAULT_MODEL_CONFIG_KWARGS: Dict[str, Any] = {"classifier_dropout": None}
+    DEFAULT_LOGGING_KWARGS: Dict[str, Any] = {
+        "use_tensorboard": True,
+        "use_wandb": True,
+        "use_csv": True,
+    }
+    DEFAULT_EARLY_STOPPING_KWARGS: Dict[str, Any] = {
+        "monitor": "val/Loss",
+        "mode": "min",
+        "patience": 3,
+    }
+
     def __init__(
         self,
         datamodule: BaseDataModule[Data],
         model: Model[BaseDataModule[Data], ModelResult],
         evaluator: Evaluator[ModelResult, EvaluationResult],
+        output_path: Path,
+        **kwargs: Any
```

Comment:
what's the use of these kwargs?

Reply:
removed

Reply:
It is a part of the workaround previously described. It allows passing `logging_kwargs` in embeddings/pipeline/lightning_hps_pipeline.py:122